sql/pre_install/tables.sql

```diff
@@ -195,8 +195,14 @@ CREATE TABLE IF NOT EXISTS _timescaledb_config.bgw_job (
     job_type            NAME        NOT NULL,
     schedule_interval   INTERVAL    NOT NULL,
     max_runtime         INTERVAL    NOT NULL,
-    max_retries         INT         NOT NULL,
+    max_retries         INTEGER     NOT NULL,
     retry_period        INTERVAL    NOT NULL,
+    proc_name           NAME        NOT NULL DEFAULT '',
+    proc_schema         NAME        NOT NULL DEFAULT '',
+    owner               NAME        NOT NULL DEFAULT CURRENT_ROLE,
+    hypertable_id       INTEGER     NOT NULL DEFAULT 0,
+    scheduled           BOOL        NOT NULL DEFAULT true,
```

Comment:
a) what does scheduled signify?  Why do we need this new column?

Reply:
scheduled means the job will be started by the scheduler, you can exempt a job from scheduling by setting this to false